src/title/TitleStream.js

```diff
@@ -70,4 +104,11 @@ const TitleStream = React.memo<Props>((props: Props) => {
 
 export default connect<SelectorProps, _, _>((state, props) => ({
   stream: getStreamInNarrow(state, props.narrow),
+  backgroundData: {
+    auth: getAuth(state),
+    mute: getMute(state),
+    subscriptions: getSubscribedStreams(state),
```

Comment:
I notice that `backgroundData.subscriptions` is `getSubscribedStreams(state)`, here, whereas in `MessageList`, it's `getSubscriptions(state)`.

I haven't yet worked out any differences in behavior this might cause, but I was wondering if this was intentional and if you had a prior feeling about which one is more correct in either place.

Reply:
Hmmm, interesting. I don't think there's a point in using `getSubscribedStreams`, so I've switched to `getSubscriptions`.